collective/portlet/content/contentportlet.pt

```diff
@@ -44,16 +47,6 @@
           Jan 10, 2010
         </p>
 
-        <div class="item-image"
-             tal:define="image view/image|nothing"
-             tal:condition="image">
-          <a tal:attributes="href view/content/absolute_url;"
-             tal:omit-tag="python:view.data.title_display != u'link'"
-             href="">
-            <img tal:replace="structure image/tag" />
-          </a>
-        </div>
```

Comment:
Seems weird that image is removed here and in the renderer, but still in the item_display vocabulary. If intentional, should that be mentioned in the history?